src/renderer/components/table/table.mixins.scss

```diff
@@ -25,7 +25,7 @@
   flex: 0 0 55px !important;
 
   > * {
-    margin: floor($margin / 1.3) auto;
+    margin: $margin * 0.7692 auto; // floor($margin / 1.3)
```

Comment:
Maybe just `margin: 6px auto`? But can be left for now I guess.

Reply:
OK, edited https://github.com/lensapp/lens/pull/3783/commits/7992521b519fb73cf2ad1bd6499f1763f9dce2d1